Library/Homebrew/formula_installer.rb

```diff
@@ -368,13 +414,14 @@ def install_fetch_deps
     end
   end
 
+  sig { void }
   def build_bottle_preinstall
-    @etc_var_dirs ||= [HOMEBREW_PREFIX/"etc", HOMEBREW_PREFIX/"var"]
-    @etc_var_preinstall = Find.find(*@etc_var_dirs.select(&:directory?)).to_a
+    @etc_var_preinstall = Find.find(*ETC_VAR_DIRS.select(&:directory?)).to_a
   end
 
+  sig { void }
   def build_bottle_postinstall
-    @etc_var_postinstall = Find.find(*@etc_var_dirs.select(&:directory?)).to_a
+    @etc_var_postinstall = Find.find(*ETC_VAR_DIRS.select(&:directory?)).to_a
```

Comment:
i'm also going to change this from an ivar → lvar (it isn't referenced outside this method)